### ISSUE TITLE
variables names containing dots are not allowed

### ISSUE THREAD

**Issue by francescolosterzo** [raphaelvallat/pingouin#107]
Hello!

I am using `pingouin` to make repeated measurements correlations between variables, and in my DataFrame some of the column names contain dots, e.g. `var.abc`.

```
pingouin.plot_rm_corr(df, x='varx.abc', y='vary.abc', subject='subject')
```
gives this error

```
NameError: name 'vary' is not defined

The above exception was the direct cause of the following exception:

PatsyError                                Traceback (most recent call last)
<ipython-input-51-764a45620ea6> in <module>
----> 1 pingouin.plot_rm_corr(df, x='varx.abc', y='vary.abc', subject=partColName)

[a lot of complaining]

~/anaconda3/envs/testenv/lib/python3.7/site-packages/patsy/compat.py in <module>

PatsyError: Error evaluating factor: NameError: name 'vary' is not defined
    vary.abc ~ C(participantNo) + varx.abc
    ^^^^^^^^
```
All in all for me was a rather quick fix, as I just replaced "." with "_" in the column names and everything worked fine, but I was wondering how complicated would it be to fix it on your side?

**Comment by raphaelvallat** [comment]
Hi @francescolosterzo !

Can you please update to the latest version of Pingouin? I've fixed this issue in version 0.3.2 with the following lines:

```
# Fit ANCOVA model
# https://patsy.readthedocs.io/en/latest/builtins-reference.html
# C marks the data as categorical
# Q allows to quote variable that do not meet Python variable name rule
# e.g. if variable is "weight.in.kg" or "2A"
formula = "Q('%s') ~ C(Q('%s')) + Q('%s')" % (y, subject, x)
model = ols(formula, data=data).fit()
```

Let me know if that solved the issue
Thanks
Raphael

**Comment by francescolosterzo** [comment]
Ooooops, I was using version 0.3.1! I updated to the latest version and it works!!
Thanks and sorry for the noise :\
